databricks_cli/stack/api.py

```diff
@@ -0,0 +1,336 @@
+# Databricks CLI
+# Copyright 2018 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+from datetime import datetime
+import time
+
+import click
+
+from databricks_cli.jobs.api import JobsApi
+from databricks_cli.version import version as CLI_VERSION
+from databricks_cli.stack.exceptions import StackError
+
+MS_SEC = 1000
+STACK_STATUS_INSERT = 'deployed'
```

Comment:
Move this into the function where it is used? Since it is just a key word in the file name

Reply:
okay sure.